docs/products/mysql/howto/connect-using-mysqlx-with-python.rst

```diff
@@ -0,0 +1,61 @@
+Connect to MySQL using MySQLx with Python
+=========================================
+
+Enabling MySQLx protocol support allows customers to use their MySQL instance as a document store. This example shows how to connect to your Aiven for MySQL instance using MySQLx protocol.
+
+.. note::
+
+    MySQL initially provided support for X-DevAPI (MySQLx) in v5.7.12 as an optional extension that can be installed by the user. On the MySQL v8.0+, the X-DevAPI is supported by default.
+
+Variables
```

Comment:
I'm a bit concerned that we put variables before we explain that they should be taken from Mysqlx tab. Should variables section go below Prerequisites?

also in Prerequisits you mention to copy SERVICE_URI, (and we removed the port), but USER, password should be copied as well. maybe it will be more clear if you use

"An Aiven account with a service running. Click on MySQLx tab under Connection information, and copy service URI, username and password needed to access the service with the help of MySQLx. They are used below as variables"  and move variables after that section

Reply:
I understand, how about create a separate Issue that maybe we can work on that to changes the order between variables and prerequisites in similar cases like the Java and Python code for this section? I think is better to choose a general format

Reply:
I can also change here in a singular file, but just want to make sure would be okay to not have a standard through all the code samples like a best option since they show up in a summary way

Reply:
This is a good point, I put the explanation of how to get the variables in the table with the variables, rather than here in the pre-requisites section (we don't normally cover the variables again here, I don't think)